### PR TITLE
A: kayak.com

### PR DESCRIPTION
**Pull request by toddhgardner**
Block a tracking agent on kayak.com from loading and sending data. #345

### CODE REVIEW

File: enhancedstats-addon.txt [hunk 1]
```diff
@@ -212,6 +212,7 @@
 /pubsys.js
 /reporting.do?
 /resonance.js
+/rollbar
```
Comment (SMed79) [inline enhancedstats-addon.txt]: bad filter, slow and too generics.

Comment (toddhgardner) [reply enhancedstats-addon.txt]: how would you recommend blocking both `rollbar.min.js` and `rollbar.js` as files?

Comment (SMed79) [reply enhancedstats-addon.txt]:
```
/rollbar.js
/rollbar.min.js
```
Filters must be very precisely and efficient as possible https://easylist.to/pages/policy.html

Comment (toddhgardner) [reply enhancedstats-addon.txt]: Thank you! Updated to match your recommendations

**Comment by toddhgardner** [comment]
@smed79 hey, is there anything else you'd like me to change with this?

**Comment by smed79** [comment]
I am not the repo owner and not an [EasyList authors](https://forums.lanik.us/memberlist.php?mode=group&g=58).

@ryanbr fyi.

**Comment by peace2000** [comment]
That site does not have any references to rollbar anymore.

**Pull request closed without merging**